docs/platform-services/threat-intelligence-indicators.md

```diff
@@ -428,7 +440,7 @@ If uploading a CSV file using the API, the file should be contained in a JSON ob
 ```
 {
  "csv": "0001,192.0.2.0,ipv4-addr,FreeTAXII,2023-02-21T12:00:00.00Z,2025-05-21T12:00:00.00Z,3,malicious-activity,\n
```

Comment:
missing comma here too, should be 
`"csv": "0001,192.0.2.0,ipv4-addr,FreeTAXII,2023-02-21T12:00:00.00Z,2025-05-21T12:00:00.00Z,3,malicious-activity,,\n`

Reply:
Thanks, @pmontiel-sumo. It's fixed now.